src/main/java/io/strimzi/kafka/bridge/quarkus/RestBridge.java

```diff
@@ -549,7 +566,47 @@ private boolean isReady() {
         return this.isReady;
     }
 
-    private void startInactiveConsumerDeletionTimer(Long timeout) {
-        // TODO: to be used a Quarkus timer for scheduling the task
+    private void jobScheduler(long timeout) throws SchedulerException {
+        long timeInMs = timeout * 1000L;
+        JobDetail job = JobBuilder.newJob(MyJob.class).build();
+        Trigger trigger = TriggerBuilder.newTrigger()
+                .withIdentity("myTrigger", "myGroup")
+                .startNow()
+                .withSchedule(
+                        SimpleScheduleBuilder.simpleSchedule()
+                                .withIntervalInMilliseconds(timeInMs / 2)
+                                .repeatForever())
+                .build();
+        quartz.scheduleJob(job, trigger);
+    }
+
+    void performTask() {
+        long timeoutInMs = httpConfig.timeoutSeconds() * 1000L;
+
+        log.infof("Looking for stale consumers in %s entries", timestampMap.size());
+        Iterator<Map.Entry<ConsumerInstanceId, Long>> it = timestampMap.entrySet().iterator();
+        while (it.hasNext()) {
+            Map.Entry<ConsumerInstanceId, Long> item = it.next();
+            if (item.getValue() + timeoutInMs < System.currentTimeMillis()) {
+                RestSinkBridgeEndpoint<byte[], byte[]> deleteSinkEndpoint = httpBridgeContext.getHttpSinkEndpoints().get(item.getKey());
+                if (deleteSinkEndpoint != null) {
+                    deleteSinkEndpoint.close();
+                    httpBridgeContext.getHttpSinkEndpoints().remove(item.getKey());
+                    log.infof("Consumer %s deleted after inactivity timeout.", item.getKey());
+                    timestampMap.remove(item.getKey());
+                }
+            }
+        }
+    }
+
+    public static class MyJob implements Job {
```

Comment:
I guess we need a more explicit name, not just `MyJob`, or?
Even adding some Javadoc would help.